### PR TITLE
pedalmarking (v5)

### PR DESCRIPTION
**Pull request by rvilarl**
PedalMarking ported with visual differences.

### CODE REVIEW

File: src/pedalmarking.ts [hunk 1]
```diff
@@ -19,11 +18,11 @@ function L(...args: any[]) {
...
  * coordinate shifts.
  */
-function drawPedalGlyph(name: string, context: RenderContext, x: number, y: number, point: number): void {
-  const glyphData = PedalMarking.GLYPHS[name];
-  const glyph = new Glyph(glyphData.code, point, { category: 'pedalMarking' });
-  // Center the middle of the glyph with the middle of the note head (Tables.STAVE_LINE_DISTANCE / 2)
-  glyph.render(context, x - (glyph.getMetrics().width - Tables.STAVE_LINE_DISTANCE) / 2, y);
+function drawPedalGlyph(name: string, ctx: RenderContext, x: number, y: number): void {
+  const glyph = new Element(PedalMarking.CATEGORY);
+  glyph.setText(PedalMarking.GLYPHS[name] || name);
```
Comment (ronyeh) [inline src/pedalmarking.ts:23]: Effectively the same, but I think we should use `??` operator when we are checking for undefined / null. We should keep `||` for actual boolean expressions (e.g., in an if / else if condition).

**Comment by ronyeh** [comment]
Yes, we might as well change the test case to use normal weight. Bold looks too heavy. Thanks for catching the bug!

**Comment by ronyeh** [comment]
OK, I can merge this after the two small changes.

File: src/pedalmarking.ts (hunk 1 above)
Comment (rvilarl) [reply src/pedalmarking.ts]: done

**Comment by rvilarl** [comment]
current
pptr-PedalMarking Custom_Text_2 Bravura svg_current
reference
pptr-PedalMarking Custom_Text_2 Bravura svg_reference

@ronyeh ready to review,/merge. Now that I see  that, the reference was probably bold.

**Pull request merged**